### ISSUE TITLE
Dante2 editor will not load

### ISSUE THREAD

**Issue by technowar** [michelson/dante2#109]
So I have this `div`,

> ```<div id="editor"></div>```

I then load the needed JS files,

> ```<script src="https://cdn.jsdelivr.net/npm/Dante2@0.4.4/dist/DanteStyles.min.js"></script>```
> ```<script src="https://cdn.jsdelivr.net/npm/Dante2@0.4.4/dist/dante-vendors.min.js"></script>```
> ```<script src="https://cdn.jsdelivr.net/npm/Dante2@0.4.4/dist/Dante2.min.js"></script>```
> ```<script src="/assets/js/editor.js"></script>```

And the CSS file,

>```<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/Dante2@0.4.4/dist/DanteStyles.min.css">```

In which, `editor.js` contains

```
(function() {
  new Dante({
    el: '#editor',
    spellcheck: true,
    disable_title: true,
    body_placeholder: 'Your Story'
  });
})();
```

In my browser's console, I see these message,

> ```init editor Dante!    dante.js:25```
 
and

> ```<SOME DANTE OPTION>    dante.js:31```

However, I am not seeing the editor. The website doesn't support `ES6` nor `npm` nor `yarn` so I added the scripts above manually.

**Comment by technowar** [comment]
Yes, I have. Before trying Dante2, I managed to make Dante (first version) work.

**Comment by michelson** [comment]
you are missing the render() function. so `var ed = new Dante({...}) ;
ed.render()`

I will suggest that use a environment to develop dante like
`create-react-app` and the compile the product to be included in your
static html site

regards

**Comment by technowar** [comment]
Now I got this error message,

```
Uncaught Error: Minified React error #200; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=200 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at r (react-dom.production.min.js:12)
    at de (react-dom.production.min.js:254)
    at Object.render (react-dom.production.min.js:255)
    at t.value (dante.js:293)
    at editor.js:11
    at editor.js:12```

**Comment by technowar** [comment]
Will close this now.
